configure.ac

```diff
@@ -1775,21 +1775,13 @@ fi
 #
 # First, determine whether we are/can support the language bindings
 #
-# Since F90/F90FLAGS are replaced by FC/FCFLAGS, rather than silently
-# substituting them, i.e. FC=$F90 and FCFLAGS=$F90FLAGS, we choose to emit
-# an error message and abort to avoid any ambiguous/hidden bug in choosing
-# Fortran90 compilers.
-if test -n "$F90" -o -n "$F90FLAGS" ; then
-    AC_MSG_ERROR([F90 and F90FLAGS are replaced by FC and FCFLAGS respectively in this configure, please unset F90/F90FLAGS and set FC/FCFLAGS instead and rerun configure again.])
-fi
-# ----------------------------------------------------------------------------
```

Comment:
I don't feel great about just ignoring the users settings vs. throwing an error. These checks are a low cost way to filter out some user issues before they become mailing list or GH issues.

Reply:
The referenced issue is complaining about this checking being over-zealous, right? From user's point of view (or my point of view as a user), it is kind of strange that a library will complain (and error-abort) on flags that it doesn't use. After all, MPICH doesn't own the namespace of `F90FLAGS`.

Reply:
Hm, a good point. A warning will likely be missed if we downgrade the check. I guess I am on board.